inc/modules/export/class-export.php

```diff
@@ -523,6 +523,8 @@ static function getExportFolder() {
 			\Pressbooks\Utility\put_contents( $path_to_htaccess, "deny from all\n" );
 		}
 
+		$path = apply_filters( 'pb_get_export_folder', $path );
```

Comment:
Add 

```
/**
 * @since 5.3.0
 *
 * TODO: Some explination as to why a user would use this
 *
 * @param string $path
 */
```

Replace the TODO up there.